sopel/config/core_section.py

```diff
@@ -389,57 +564,86 @@ def homedir(self):
 
     :default: ``chat.freenode.net``
 
-    **Required.**
+    **Required**:
+
+    .. code-block:: ini
+
+        host = chat.freenode.net
+
     """
 
     host_blocks = ListAttribute('host_blocks')
     """A list of hostnames which Sopel should ignore.
 
-    Messages from any user whose connection hostname matches one of these values
-    will be ignored. :ref:`Regular expression syntax <re-syntax>` is supported.
+    Messages from any user whose connection hostname matches one of these
+    values will be ignored. :ref:`Regular expression syntax <re-syntax>` is
+    supported:
```

Comment:
Please don't re-wrap this. It fits perfectly on two lines if the text is allowed to be 80 columns instead of 79.